guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -28,7 +17,8 @@ ifndef::orcharhino[]
 . In the *Repository Sets* tab, ensure only your named repository is enabled.
```

Comment:
I feel like this was left over from when custom products were enabled by default. In any case, I think this step could be removed. Or, we could just say more generally

```
. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired. For more information, see xxx
```